### PR TITLE
Map some more DataFixer code.

### PR DESCRIPTION
**Pull request by i509VCB**

### CODE REVIEW

**Review by liach — APPROVED** [review]

File: mappings/net/minecraft/datafixer/NbtOps.mapping [hunk 1]
```diff
@@ -1,2 +1,3 @@
 CLASS net/minecraft/class_2509 net/minecraft/datafixer/NbtOps
+	COMMENT Used to represent Minecraft's nbt tags as a {@link com.mojang.datafixers.Dynamic} for DataFixerUpper.
```
Comment (YanisBft) [inline mappings/net/minecraft/datafixer/NbtOps.mapping]: > Minecraft's nbt tags

It means "non binary tags tags"

Comment (liach) [reply mappings/net/minecraft/datafixer/NbtOps.mapping]:
```suggestion
	COMMENT Used to represent a Minecraft NBT as a {@link com.mojang.datafixers.Dynamic} for DataFixerUpper.
```

**Review by YanisBft — APPROVED** [review]

File: mappings/net/minecraft/datafixer/fix/BlockEntityCustomNameToTextFix.mapping [hunk 2]
```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_3567 net/minecraft/datafixer/fix/BlockEntityCustomNameToComponentFix
+CLASS net/minecraft/class_3567 net/minecraft/datafixer/fix/BlockEntityCustomNameToTextFix
```
Comment (Juuxel) [inline mappings/net/minecraft/datafixer/fix/BlockEntityCustomNameToTextFix.mapping:1]: This name is given by a string in `makeRule`, so it should be left the way it is.

File: mappings/net/minecraft/datafixer/fix/EntityCustomNameToTextFix.mapping [hunk 3]
```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_3597 net/minecraft/datafixer/fix/EntityCustomNameToComponentFix
+CLASS net/minecraft/class_3597 net/minecraft/datafixer/fix/EntityCustomNameToTextFix
```
Comment (Juuxel) [inline mappings/net/minecraft/datafixer/fix/EntityCustomNameToTextFix.mapping:1]: Same here.

**Review by Juuxel — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/datafixer/fix/BlockEntityCustomNameToTextFix.mapping (hunk 2 above)
Comment (i509VCB) [reply mappings/net/minecraft/datafixer/fix/BlockEntityCustomNameToTextFix.mapping:1]: My reason for the change is that we don't use Component as a name anymore

Comment (liach) [reply mappings/net/minecraft/datafixer/fix/BlockEntityCustomNameToTextFix.mapping:1]: I support i509VCB here.

**Comment by Juuxel** [comment]
(got some inspiration from #yarn-triage)

I think it would be fine if the classes were renamed with a comment like "This fix is internally called `MojangNameHereFix`".

**Comment by liach** [comment]
Is a javadoc comment still necessary if there is string name?

**Review by Runemoro — APPROVED** [review]

**Pull request merged**